### ISSUE TITLE
Make library work more like UIAlertView

### ISSUE THREAD

**Issue by arronhunt** [JSSAlertView/JSSAlertView#73]
Making the library work more like the default UIAlertView would enable you to further customize the alert options, such as:

* Moving the options to the constructor `JSSAlertView(self, title: "I'm an alert").show()`
* Allow for multiple buttons to be added, via `let myButton = JSSAlertButton(title: 'Click me') { action in NSLog('Clicked a button')` and then adding them via `myJSSAlertView.addButton(myButton)`

Just my 2 cents :)

**Comment by syky27** [comment]
Yeah, I was thinking about this for some time, unfortunately I do not currently have the time to do that...
